### PR TITLE
Alice's Hello

### PR DESCRIPTION
**Pull request by ellst029**
This is my hello! Hello!

### CODE REVIEW

**Review by JadynSondrol — DISMISSED** [review]

**Review by JacobJenness — DISMISSED** [review]
Line 21 should have the following.
`builder.append(aliceSaysHello());`
This will make sure that `Hellos.java` will print your greeting.

File: src/main/java/hellos/Hellos.java [hunk 1]
```diff
@@ -35,9 +36,12 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+alice-greeting
```
Comment (floogulinc) [inline src/main/java/hellos/Hellos.java]: You probably didn't mean to leave this in from merging!
```suggestion
```

**Comment by NicMcPhee** [comment]
If things get weird, I would definitely recommend pulling the latest version of your branch to your computer, and then working on things in VS Code. You can run the tests there, and it will check that your code compiles, etc.

File: src/main/java/hellos/Hellos.java [hunk 2]
```diff
@@ -19,6 +19,7 @@ public void run() {
   public String generateOutput() {
     StringBuilder builder = new StringBuilder();
 
+    builder.append(
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: This isn't a complete statement/line. You probably wanted:

```suggestion
    builder.append(aliceSaysHello());
```

File: src/main/java/hellos/Hellos.java [hunk 3]
```diff
@@ -35,9 +36,12 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+alice-greeting
+  private String aliceSaysHello(){
```
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: The "standard" is to have a space before the curly brace at the start of a method:

```suggestion
  private String aliceSaysHello() {
```

**Review by NicMcPhee — CHANGES_REQUESTED** [review]

**Review by NicMcPhee — APPROVED** [review]
That looks great – thanks for fixing those compiler errors.

**Review by halleypaulson — APPROVED** [review]

**Pull request merged**